### PR TITLE
feat: List and Modal

### PR DESCRIPTION
**Pull request by manish-GH**
Added List and Modal components

List: https://brave-shirley-b5c994.netlify.app/components/list/
Modal: https://brave-shirley-b5c994.netlify.app/components/modal/

### CODE REVIEW

File: components/Modal/styles.css [hunk 1]
```diff
@@ -1 +1,55 @@
...
+
+.modal-container {
+	z-index: 999;
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100vw;
+	height: 100vh;
+	display: none;
+	align-items: center;
```
Comment (sanjitsarkar) [inline components/Modal/styles.css]: If you want to center a div you can use this concise syntax 
display:grid;
place-content:center;

File: components/Modal/script.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const modalBtn = document.querySelector("#display-modal");
+const xBtn = document.querySelector("#x-btn");
```
Comment (sanjitsarkar) [inline components/Modal/script.js]: Use better and meaningful name for variable like you can name this variable to closeButton or closeBtn

**Review by sanjitsarkar — APPROVED** [review]

File: components/Modal/styles.css (hunk 1 above)
Comment (manish-GH) [reply components/Modal/styles.css]: Ohh, thanks for this

**Comment by amantulsyan35** [comment]
apart from the changes @sanjitsarkar  suggested, looks good to me

**Pull request merged**